engine/security-api/src/main/scala/pl/touk/nussknacker/ui/security/api/LoggedUser.scala

```diff
@@ -1,15 +1,17 @@
 package pl.touk.nussknacker.ui.security.api
 
-import Permission.Permission
+import pl.touk.nussknacker.ui.security.api.Permission.Permission
 
-case class LoggedUser(id: String,
-                      categoryPermissions: Map[String, Set[Permission]]=Map.empty) {
+case class LoggedUser(
+  id: String,
+  categoryPermissions: Map[String, Set[Permission]] = Map.empty,
+  isAdmin: Option[Boolean] = Option.apply(false)
```

Comment:
if we have default value does it make sense to have option here?

Reply:
Good idea :)